lld/COFF/Chunks.cpp

```diff
@@ -411,7 +411,9 @@ void SectionChunk::writeTo(uint8_t *buf) const {
     applyRelocation(buf + rel.VirtualAddress, rel);
   }
 
-  // Write the offset to EC entry thunk preceding section contents.
+  // Write the offset to EC entry thunk preceding section contents. The low bit
+  // is always set, so it's effectively an offset from the last byte of the
+  // padding.
```

Comment:
Maybe `offset` instead of `padding` here?

Reply:
I changed it and pushed. Thanks!